ee-galleon-pack/pom.xml

```diff
@@ -3978,6 +3978,11 @@
             </exclusions>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>xml-resolver</groupId>
+            <artifactId>xml-resolver</artifactId>
+ 	    <scope>provided</scope>
```

Comment:
There's something odd here with whitespace. When I use the View File option and see the whole file these line up. Perhaps there is a mix of tabs and spaces here?

Reply:
May be, I will check all the comments and update accordingly.